drivers/source/SerialBase.cpp

```diff
@@ -164,6 +166,28 @@ void SerialBase::_deinit()
     serial_free(&_serial);
 }
 
+int SerialBase::sync()
+{
+    int count = 0;
+    // See send_break(), ensure at least 1ms sleep time
+    int timeout_ms = (18000 / _baud) + 1;
```

Comment:
Still not figured out what I think about the timeout principle, but while I figure it out, keep it the Chrono domain, rather than having an `int`.

    auto char_time_allowance = milliseconds_u32((18000 / _baud) + 1);

    sleep_for(char_time_allowance);

Or some similar name - it's not the actual timeout.

Reply:
Sure! Fixed with 66ecb29ed0e2657e792f71c91d440d89e892a1f3.